### PR TITLE
Add workflow step that shows build scan url

### PR DESCRIPTION
**Pull request by laurit**

### CODE REVIEW

**Review by zeitlinger — APPROVED** [review]
great!

File: .github/workflows/build-common.yml [hunk 1]
```diff
@@ -267,6 +267,14 @@ jobs:
           # gradle enterprise is used for the build cache
           gradle-home-cache-excludes: caches/build-cache-1
 
+      - id: read-build-scan
+        name: Read build scan
+        if: ${{ hashFiles('build-scan.txt') != '' }}
+        run: echo "build-scan=$(cat build-scan.txt)" >> $GITHUB_OUTPUT
+      - name: Build scan
+        if: ${{ hashFiles('build-scan.txt') != '' }}
+        run: echo ${{ steps.read-build-scan.outputs.build-scan }}
```
Comment (trask) [inline .github/workflows/build-common.yml:276]: why 2 steps instead of 1?

**Review by trask — APPROVED** [review]

File: .github/workflows/build-common.yml (hunk 1 above)
Comment (laurit) [reply .github/workflows/build-common.yml:276]: The sample code I found from google used 2 steps. I was intending on relying on you pointing out a simpler solution if one exists.

Comment (zeitlinger) [reply .github/workflows/build-common.yml:276]: maybe the $GITHUB_OUTPUT is displayed in some summary box?

**Review by jeanbisutti — APPROVED** [review]

File: .github/workflows/build-common.yml (hunk 1 above)
Comment (trask) [inline .github/workflows/build-common.yml:276]: I think this should work, I'll accept the suggestion and see...

```suggestion
      - name: Build scan
        if: ${{ hashFiles('build-scan.txt') != '' }}
        run: cat build-scan.txt
```

File: .github/workflows/reusable-test-indy.yml [hunk 2]
```diff
@@ -85,3 +85,11 @@ jobs:
...
           # gradle enterprise is used for the build cache
           gradle-home-cache-excludes: caches/build-cache-1
+
+      - id: read-build-scan
+        name: Read build scan
+        if: ${{ hashFiles('build-scan.txt') != '' }}
+        run: echo "build-scan=$(cat build-scan.txt)" >> $GITHUB_OUTPUT
+      - name: Build scan
+        if: ${{ hashFiles('build-scan.txt') != '' }}
+        run: echo ${{ steps.read-build-scan.outputs.build-scan }}
```
Comment (trask) [inline .github/workflows/reusable-test-indy.yml:95]:
```suggestion
      - name: Build scan
        if: ${{ hashFiles('build-scan.txt') != '' }}
        run: cat build-scan.txt
```

File: .github/workflows/reusable-test-latest-deps.yml [hunk 3]
```diff
@@ -83,6 +83,9 @@ jobs:
           # gradle enterprise is used for the build cache
           gradle-home-cache-excludes: caches/build-cache-1
 
+      - name: Build scan
+        if: ${{ hashFiles('build-scan.txt') != '' }}
+        run: cat build-scan.txt
```
Comment (trask) [inline .github/workflows/reusable-test-latest-deps.yml:88]:
```suggestion
        run: cat build-scan.txt

```

**Comment by tylerbenson** [comment]
Thanks for adding this @laurit!

**Pull request merged**